src/Components/Pages/ItemDetailsPage.js

```diff
@@ -0,0 +1,31 @@
+import {Button, Image, Container} from 'react-bootstrap';
+import Navigation from '../Layout/Navigation';
+import image from '../Images/1984first.jpg';
+import '../Page Styles/ItemDetailsPage.css'
+
+function ItemDetailsPage() {
+    return (
+        <><Navigation />
```

Comment:
The outermost `<>` can be in a separate line.

You can use `shift+alt+f` to auto indent a file, before pushing it to origin.

But it doesn't really matter. Let's keep it like this.